.github/dependabot.yml

```diff
@@ -5,10 +5,6 @@ updates:
     schedule:
       interval: "daily"
       time: "11:00"
-    assignees:
-      - "jawnsy"
-    reviewers:
-      - "jawnsy"
```

Comment:
There's also another entry under npm

thanks for taking care of this! :heart:

Reply:
Oops! Fixing that too! Thanks for catching that!